app/javascript/tests/mocks/mockMonitoringReasons.js

```diff
@@ -0,0 +1,21 @@
+export default [
```

Comment:
Shouldn't this mock include `'', nil`?

Reply:
Those are added in the UI, so they cant be added to the master list (otherwise you'd get a duplication). To prevent this, on line 62 of `MonitoringStatus.test.js` we have ` [''].concat(mockMonitoringReasons).forEach(function(value, index) {`